modules/@shopify/checkout-sheet-kit/src/index.d.ts

```diff
@@ -74,70 +75,95 @@ export interface AndroidAutomaticColors {
   dark: AndroidColors;
 }
 
-export type Configuration =
-  | {
-      /**
-       * The selected color scheme for the checkout. See README.md for more details.
-       */
-      colorScheme?: ColorScheme.web | ColorScheme.light | ColorScheme.dark;
-      /**
-       * Enable/disable preloading for checkout. This option must be enabled for `.preload()` to work as expected.
-       */
-      preloading?: boolean;
-      /**
-       * Platform-specific color overrides
-       */
-      colors?: {
-        ios?: IosColors;
-        android?: AndroidColors;
-      };
-    }
-  | {
-      /**
-       * The selected color scheme for the checkout. See README.md for more details.
-       */
-      colorScheme?: ColorScheme.automatic;
-      /**
-       * Enable/disable preloading for checkout. This option must be enabled for `.preload()` to work as expected.
-       */
-      preloading?: boolean;
-      /**
-       * Platform-specific color overrides
-       */
-      colors?: {
-        ios?: IosColors;
-        android?: AndroidAutomaticColors;
-      };
-    };
+interface CommonConfiguration {
+  /**
+   * Enable/disable preloading for checkout. This option must be enabled for `.preload()` to work as expected.
+   */
+  preloading?: boolean;
+  /**
+   * Sets the title of the Checkout sheet.
+   *
+   * * Important: This will only modify the Checkout Sheet on iOS, not Android.
+   *
+   * To implement localization support for iOS:
+   *  1. Create a "Localizable.xcstrings" file under "ios/{YourApplication}"
+   *  2. Set a translated value for a "shopify_checkout_sheet_title" key
+   *
+   * To implement localization support for Android:
+   *  1. Open the "android/app/src/main/res/values/strings.xml" file
+   *  2. Add "<string name="checkout_web_view_title">Checkout</string>"
+   */
+  title?: string;
+}
+
+export type Configuration = CommonConfiguration &
+  (
+    | {
+        /**
+         * The selected color scheme for the checkout. See README.md for more details.
+         */
+        colorScheme?: ColorScheme.web | ColorScheme.light | ColorScheme.dark;
```

Comment:
I think I might have forgotten, why can't we do 

`colorScheme?: ColorScheme.web | ColorScheme.light | ColorScheme.dark | ColorScheme.automatic;`

Reply:
For Android, when the consumer is using `automatic`, they can set the colors for both variants. So the `colors` attribute changes depending on the `colorScheme`. I had to union them for this reason